spec/beaker/ssh_connection_spec.rb

```diff
@@ -1,6 +1,22 @@
 require 'spec_helper'
 require 'net/ssh'
 
+module Net
+  module SSH
+    describe Buffer do
+      it 'has a monkey patch tested on 2.9.4' do
```

Comment:
This is a guard to make sure we notice when a new version releases - requiring us to verify the patch still makes sense.